src/js/main.js

```diff
@@ -47,30 +48,144 @@ Site = {
 };
 
 Site.Overlay = {
+  activeNavItem: null,
+  desktopWidth: 1024,
+
   init: function() {
     var _this = this;
 
+    // GET! ALL THE THINGS!
+    _this.pageOverlay = document.getElementsByClassName('page-overlay');
+
+    _this.header = document.getElementById('header');
+
     _this.mobileToggle = document.getElementById('mobile-overlay-toggle');
     _this.mobileMenu = document.getElementById('mobile-menu');
 
-    _this.bindToggles();
+    _this.pageOpen = document.getElementsByClassName('js-page-open');
+
+    _this.desktopPageClose = document.getElementsByClassName('desktop-page-close');
+
+    // BIND CLICKS!
+    _this.bindClicks();
   },
 
-  bindToggles: function() {
+  bindClicks: function() {
     var _this = this;
 
-    _this.mobileToggle.addEventListener('click', _this.handleToggle.bind(_this));
+    // bind the mobile toggle clicks!
+    _this.mobileToggle.addEventListener('click', _this.toggleMobile.bind(_this));
+
+    for (var i = 0; i < _this.pageOpen.length; i++) {
```

Comment:
This really is unnecessary when we have jQuery. You cannot claim (even @mixcas can't surely) that this is more readable, quicker to write, cleaner than writing `$(item).on('click', function() {});` Please can we not go it in the backwards direction.